xayagametest/testcase.py

```diff
@@ -83,6 +85,14 @@ def main (self):
 
     randomSuffix = "%08x" % random.getrandbits (32)
     self.basedir = os.path.join (self.args.dir, DIR_PREFIX + randomSuffix)
+    
+    if sysconfig.get_platform() == "mingw":
+      self.args.dir = tempfile.gettempdir()
+      self.basedir = os.path.join (self.args.dir, DIR_PREFIX + randomSuffix)
+      spath = sys.path[0]
+      spath = spath.replace("\\","/")
+      self.args.game_daemon = spath + "/../src/" + self.gameId + "d.exe"     
```

Comment:
This is a hack that is not guaranteed to work with every game.  Instead, the game's test class should pass the right binary as `gameBinaryDefault` already.